core/ui_mixins/quick_panel.py

```diff
@@ -139,24 +142,15 @@ def show_paginated_panel(items, on_done, flags=None, selected_index=None, on_hig
             limit=limit,
             next_message=next_message)
     pp.show()
-
-
-def show_log_panel(entries, on_done, limit=6000, selected_index=None):
-    """
-    Display log entries in quick panel with pagination, and execute on_done(commit)
-    when item is selected. `entries` can be either a list or a generator of LogEnty.
-
-    """
-    lp = LogPanel(entries, on_done, limit=limit, selected_index=selected_index)
-    lp.show()
+    pp
```

Comment:
I meant `return pp`. Doing julia too much earlier. In julia, the last expression always returns